sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -243,8 +243,9 @@ void gnrc_ipv6_netif_set_router(gnrc_ipv6_netif_t *netif, bool enable)
         gnrc_sixlowpan_nd_router_set_router(netif, enable);
         return;
     }
-#endif
+#else
```

Comment:
Updated in the latest commit, but can you tell me in which case a node would have both modules enabled? Border router?